trimmer_decorator.rb

```diff
@@ -0,0 +1,8 @@
+require './base_decorator'
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    return @nameable.correct_name[0..9] if @nameable.correct_name.length < 10
```

Comment:
- [ ] @AbigiyaTY great work implementing the changes requested however in this class, the class method `correct_name` will not trim a name longer than 10 characters, your statement in line four indicates it should trip to  10 characters if the length of the name is less than 10 kindly correct it as previously requested.